src/test/java/com/mentorship/userservice/repository/AuthRepositoryTest.java

```diff
@@ -0,0 +1,96 @@
+package com.mentorship.userservice.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.mentorship.userservice.domain.Auth;
+import com.mentorship.userservice.domain.User;
+import com.mentorship.userservice.dto.enums.UserRole;
+import com.mentorship.userservice.repositories.UserRepository;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+@DataJpaTest
+public class AuthRepositoryTest {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    private User user1;
+    private String user1Email;
+    private String user1Name;
+    private String user1LastName;
+    private String user1PhoneNumber;
+    private String user2Name;
+
+
+    @BeforeEach
+    public void createUsers() {
+
+        user1Email = "ivan@gmail.com";
+        String user2Email = "taras@gmail.com";
+        user1Name = "Ivan";
+        user2Name = "Taras";
+        user1LastName = "Ivanenko";
+        String user2LastName = "Tarasenko";
+        user1PhoneNumber = "+111111111";
+        User user1Object = createUserObject(user1Name, user1LastName, user1PhoneNumber, user1Email);
+        User user2Object = createUserObject(user2Name, user2LastName, user1PhoneNumber, user2Email);
+        user1 = userRepository.save(user1Object);
+        userRepository.save(user2Object);
+    }
+
+    @Test
+    public void shouldReturnTrueIfEmailPresentInDb() {
```

Comment:
usually it is needed to test custom queries, not auto generated ones. It is better to test auto-generated queries at the e2e level, when we don't mock db level, but use DB instead